lgc/builder/DescBuilder.cpp

```diff
@@ -121,9 +130,16 @@ Value *DescBuilder::CreateLoadBufferDesc(unsigned descSet, unsigned binding, Val
   }
 
   if (!desc) {
-    if (node) {
+    if (node || node->concreteType == ResourceNodeType::DescriptorMutable) {
```

Comment:
This is worse now because it might dereference a null pointer :)

Reply:
Yes apologies, should be fixed properly now